app/models/review.rb

```diff
@@ -0,0 +1,6 @@
+class Review < ApplicationRecord
```

Comment:
Indentation - here you have 2 spaces per indentation, elsewhere you have 6, it should be 4.

Set up RubyMine accordingly.